### PR TITLE
[STORM-2134] - improving the current scheduling strategy for RAS

### PR DESCRIPTION
**Pull request by jerrypeng**
Please see modified documentation for more info on what changed

### CODE REVIEW

**Comment by jerrypeng** [comment]
Also re-add some RAS related tests that were accidentally deleted in the past

File: storm-core/test/jvm/org/apache/storm/scheduler/resource/strategies/eviction/TestDefaultEvictionStrategy.java [hunk 1]
```diff
@@ -0,0 +1,736 @@
+package org.apache.storm.scheduler.resource.strategies.eviction;
```
Comment (revans2) [inline storm-core/test/jvm/org/apache/storm/scheduler/resource/strategies/eviction/TestDefaultEvictionStrategy.java]: Needs a license header.

Comment (jerrypeng) [inline storm-core/test/jvm/org/apache/storm/scheduler/resource/strategies/eviction/TestDefaultEvictionStrategy.java]: added

**Comment by jerrypeng** [comment]
@revans2 thanks for the reviews.  I have addressed your comments

**Comment by revans2** [comment]
+1

**Pull request merged**